spec/app/domain/services/update_course_exercise_exclusions/service_spec.rb

```diff
@@ -0,0 +1,155 @@
+require 'rails_helper'
+
+RSpec.describe Services::UpdateCourseExerciseExclusions::Service do
+
+  let(:service) { Services::UpdateCourseExerciseExclusions::Service.new }
+
+  let(:action_none) { service.process(
+    update_uuid:      given_update_uuid,
+    sequence_number:  given_sequence_number,
+    course_uuid:      given_course_uuid,
+    exclusions:       given_none_exclusions.exclusions
+  ) }
+
+  let(:action_some) { service.process(
+    update_uuid:      given_update_uuid,
+    sequence_number:  given_sequence_number,
+    course_uuid:      given_course_uuid,
+    exclusions:       given_some_exclusions.exclusions
+  ) }
+
+  let(:action_many) { service.process(
+    update_uuid:      given_update_uuid,
+    sequence_number:  given_sequence_number,
+    course_uuid:      given_course_uuid,
+    exclusions:       given_many_exclusions.exclusions
+  ) }
+
+  let(:action_repeat) { service.process(
+    update_uuid:      given_update_uuid,
+    sequence_number:  given_sequence_number,
+    course_uuid:      given_course_uuid,
+    exclusions:       given_some_exclusions.exclusions
+  ) }
+
```

Comment:
I feel like these should be defined in their respective contexts, not out here in the "wild".  It also seems like there could be just one (maybe two, for the repeat case?) and `given_exclusions` could be adjusted per context.